packages/expo-updates/scripts/createManifest.js

```diff
@@ -1,8 +1,10 @@
-const { loadMetroConfigAsync } = require('@expo/cli/build/src/start/server/metro/instantiateMetro');
```

Comment:
`expo-updates` doesn't have an explicit dependency on `@expo/cli`. should it?

Reply:
It should derive its `@expo/cli` version from the `expo` version, I would think

Reply:
Does `expo` depend on `@expo/cli`? (explicitly, in `package.json`)

Reply:
Yes indeed, see `packages/expo/package.json`